app/api/tickets.py

```diff
@@ -67,6 +69,14 @@ def query(self, view_kwargs):
         :return:
         """
         query_ = self.session.query(Ticket).filter_by(is_hidden=False)
```

Comment:
@srv-twry I included this as default behavior. Do i still need else statement after if in code below?

Reply:
I think the issue in this approach is that you will querying twice when the user is the organizer or the admin. 
first the default one and then in case when the `if` condition is executed. 
I would suggest that you `else` conditions. The code would look slightly more complicated but it won't query the db twice. Views ?

Reply:
Yes, that's right. Let me make changes.